app/views/questions/show.html.erb

```diff
@@ -1,6 +1,10 @@
 <div>
   <h1><%= @question.title %></h1>
   <%= @question.content %>
+  <% @question.comments.each do |comment| %>
+    <%= comment.content %>
+  <% end %>
+  <%= link_to 'Comment', new_comment_path(@question.id) %>

```

Comment:
Need something like this

link_to 'Comment', new_comment_path, commentable_id: 1, commentable_type: 'ClassName'